packages/e2e/playwright.config.ts

```diff
@@ -0,0 +1,62 @@
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+require("dotenv").config();
```

Comment:
not sure if it's needed, but we can validate if the required env vars were provided

Reply:
Its needed and its validated. Added comment where to look it up